### PR TITLE
1548860583 - AADS User, on the DS storybook, I can see a renderer  that is both a dropdown and a free text (CDB table and ContactPanel)

### PR DESCRIPTION
**Pull request by frank-shenton**
Before submitting my PR to Code Review, I have made sure:

- [x] I have run the tests using `yarn run test`
- [x] I have exported any new files in index.js
- [x] My file supports Formik if will potentially be used in a form
- [x] The PR name follows the convention `(#<ticket_number>) - <ticket_user_story>`
- [x] I have incremented the version in `package.json` if I want to publish a new version
- [x] If my change is breaking I have updated the minor version number(i.e. the middle one) and don't merge until I have the PR on Corazon ready

### CODE REVIEW

File: src/components/atoms/Select/Select.test.js [hunk 1]
```diff
@@ -47,63 +41,6 @@ describe("Select Input test", () => {
...
-      <Select options={{}} id="testId" isClearable customSort={customSort} />,
-    ).find("CustomSelect")
-    expect(component.render().text()).toEqual("Type to search from the dropdown list...")
-    expect(component.instance().renderOptions(options)).toEqual([
-      { value: "mouse", label: "Mouse" },
-      { value: "horse", label: "Horse" },
-      { value: "dog", label: "Dogs" },
-      { value: "cat", label: "Cats" },
-    ])
-  })
```
Comment (elio-stalteri) [inline src/components/atoms/Select/Select.test.js:106]: why is this test deleted?

File: src/components/atoms/Select/Select.test.js [hunk 2]
```diff
@@ -113,17 +50,13 @@ describe("Select Input test", () => {
 
     expect(component).toMatchSnapshot()
   })
-  it("returns the correctly formatted default value with label when there are no options", () => {
-    const component = mount(<Select options={{}} id="testId" isClearable />).find("CustomSelect")
-    const defaultValueObject = component.instance().buildDefaultValue("cat")
-    expect(defaultValueObject).toMatchObject({ value: "cat", label: "cat" })
-  })
```
Comment (elio-stalteri) [inline src/components/atoms/Select/Select.test.js:53]: why is this test deleted?

File: src/components/atoms/Select/Select.test.js (hunk 1 above)
Comment (frank-shenton) [reply src/components/atoms/Select/Select.test.js:106]: I moved renderOptions to the shared services/select file, and created new tests for it there.

File: src/components/atoms/Select/Select.test.js (hunk 2 above)
Comment (frank-shenton) [reply src/components/atoms/Select/Select.test.js:53]: Same as the other one, I moved the function to a shared location and tested it there.

**Review by elio-stalteri — APPROVED** [review]

**Pull request merged**